routes/web/project/load_balancer.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class CloverWeb
+  hash_branch(:project_prefix, "load-balancer") do |r|
+    r.get true do
+      lbs = @project.private_subnets.flat_map { |ps| ps.load_balancers_dataset.authorized(@current_user.id, "LoadBalancer:view").all }
+      @lbs = Serializers::LoadBalancer.serialize(lbs, {include_path: true})
+
+      view "networking/load_balancer/index"
+    end
+
+    r.post true do
+      Authorization.authorize(@current_user.id, "LoadBalancer:create", @project.id)
+
+      ps = PrivateSubnet.from_ubid(r.params["private-subnet-id"])
+
+      lb = Prog::Vnet::LoadBalancerNexus.assemble(
+        ps.id,
+        name: r.params["name"],
+        algorithm: r.params["algorithm"],
+        src_port: r.params["src-port"],
```

Comment:
We are migrating to underscore in parameter names. Can you change them?
https://github.com/ubicloud/ubicloud/commit/6a9cc5cdb7c9b2b800e45f7122538c3ecb46aca6

Reply:
updated